mod_form.php

```diff
@@ -68,6 +84,16 @@ public function definition() {
     }
 
     public function data_preprocessing(&$defaultvalues) {
+        $content = null;
+        if (!empty($defaultvalues['id'])) {
+            // Load Content
```

Comment:
The code two lines below already says `->loadContent` so there's no need for commenting it separately. ;)

It would be better if the comment instead explained what is being loaded, from where and why.